change/@microsoft-teams-js-e08b2a2a-5a00-4e48-9366-4595b5a723d0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
I don't think this is a major change because major means breaking change. This is probably a minor change because we're adding new functionality in a backwards compatible way